src/GenAPI/Program.cs

```diff
@@ -266,6 +268,7 @@ private static void ParseCommandLine(string[] args)
                 parser.DefineOptionalQualifier("hightlightInterfaceMembers", ref s_hightlightInterfaceMembers, "(-him) [CSDecl] Highlight interface implementation members.");
                 parser.DefineAliases("throw", "t");
                 parser.DefineOptionalQualifier("throw", ref s_throw, "(-t) Method bodies should throw PlatformNotSupportedException.");
+                parser.DefineOptionalQualifier("exceptionMessage", ref s_exceptionMessage, "PlatformNotSupportedException custom message.");
```

Comment:
Assembly granularity seems OK, but I could imagine we might want a data file that maps doc-id to message in the case a single assembly wanted different messages per type.